install_config/install/disconnected_install.adoc

```diff
@@ -320,6 +359,42 @@ and then transporting them:
     registry.access.redhat.com/openshift3/metrics-cassandra \
     registry.access.redhat.com/openshift3/metrics-heapster
 ----
++
+----
+# docker save -o ose3-logging-metrics-images.tar \
+    registry.access.redhat.com/openshift3/container-engine \
+    registry.access.redhat.com/openshift3/efs-provisioner \
+    registry.access.redhat.com/openshift3/jenkins-1-rhel7 \
+    registry.access.redhat.com/openshift3/jenkins-2-rhel7 \
+    registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7 \
+    registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7 \
+    registry.access.redhat.com/openshift3/jenkins-slave-nodejs-rhel7 \
+    registry.access.redhat.com/openshift3/logging-auth-proxy \
+    registry.access.redhat.com/openshift3/logging-curator \
+    registry.access.redhat.com/openshift3/logging-deployer \
+    registry.access.redhat.com/openshift3/logging-elasticsearch \
+    registry.access.redhat.com/openshift3/logging-fluentd \
+    registry.access.redhat.com/openshift3/logging-kibana \
+    registry.access.redhat.com/openshift3/metrics-cassandra \
+    registry.access.redhat.com/openshift3/metrics-deployer \
+    registry.access.redhat.com/openshift3/metrics-hawkular-metrics \
+    registry.access.redhat.com/openshift3/metrics-hawkular-openshift-agent \
+    registry.access.redhat.com/openshift3/metrics-heapster \
+    registry.access.redhat.com/openshift3/node \
+    registry.access.redhat.com/openshift3/openvswitch
+----
+
+*Is this needed?*
```

Comment:
ASB depends on product registry, so in a disconnected environment, from my understanding, no need to mention this part.